components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3009,6 +3051,9 @@ where
                     self.on_gc_peer_request(msg);
                 }
             }
+            ExtraMessageType::MsgTombstonePeerRequest => {
```

Comment:
Tombstone is ambiguous, it actually does remove peer

Reply:
How about `MsgForcelyRemovePeerRequest` ?

Reply:
lgtm